src/work-allocation-2/components/case-manager-filter/case-manager-filter.component.ts

```diff
@@ -1,97 +1,151 @@
+import { filter } from 'rxjs/operators';
 import { Location } from '../../models/dtos';
-import { FilterConstants } from '../constants';
 
 @Component({
   selector: 'exui-case-manager-filter',
   templateUrl: './case-manager-filter.component.html',
-  styleUrls: ['./case-manager-filter.component.scss']
+  styleUrls: ['./case-manager-filter.component.scss'],
+  encapsulation: ViewEncapsulation.None,
 })
-export class CaseManagerFilterComponent implements OnChanges {
-  // Protected instances of the exported constants.
-  public readonly ALL_LOCATIONS: Location = FilterConstants.Options.Locations.ALL;
+export class CaseManagerFilterComponent implements OnInit {
 
-  /**
-   * The locations that are available to be selected from.
-   */
-  @Input() public locations: Location[];
+  private static FILTER_NAME: string = 'cases';
 
-  @Input()
-  public get location(): Location {
-    return this.pLocation;
-  }
+  @Input() public locations: Location[] = [];
 
-  public set location(value: Location) {
-    value = value || this.ALL_LOCATIONS; // undefined or null means "All"
-    if (this.pLocation !== value) {
-      this.pLocation = value;
-      this.saveFilterToSession();
-      this.emitChangedEvent();
+  public fieldsConfig: FilterConfig = {
+    persistence: 'session',
+    enableDisabledButton: true,
+    id: CaseManagerFilterComponent.FILTER_NAME,
+    fields: [],
+    cancelButtonText: 'Reset to default',
+    applyButtonText: 'Apply',
+    cancelSetting: {
+      id: CaseManagerFilterComponent.FILTER_NAME,
+      fields: [
+        {
+          name: 'service',
+          value: ['IA']
+        }
+      ]
     }
-  }
+  };
 
-  @Input()
-  public get jurisdiction(): any {
-    return this.pJurisdiction;
-  }
+  constructor(private readonly filterService: FilterService) {
 
-  public set jurisdiction(value: any) {
-    if (this.pJurisdiction !== value) {
-      this.pJurisdiction = value;
-      this.saveFilterToSession();
-      this.emitChangedEvent();
-    }
   }
 
-  private pLocation: Location = this.ALL_LOCATIONS;
-  private pJurisdiction: any =  {id: 'IA', name: 'Immigration and Asylum'};
-
-  /**
-   * Emit an event to notify the parent component that the selected Caseworker
-   * and/or Location has changed. The new selection is emitted with the event
-   * but can also be retrieved from component.location.
-   */
-  @Output() public selectionChanged: EventEmitter<{ location: Location, jurisdiction: any}>
-    = new EventEmitter<{ location: Location, jurisdiction: any }>();
-
-  /**
-   * Accept the SessionStorageService for adding to and retrieving from sessionStorage.
-   */
-  constructor(private readonly sessionStorageService: SessionStorageService) {
+  private static initServiceFilter(): FilterFieldConfig {
+    return {
+      name: 'service',
+      options: [
+        {
+          key: 'IA',
+          label: 'Immigration and Asylum'
+        }
+      ],
```

Comment:
ditto, could this be extracted to app.constants.ts, for easier future maintainance.